### ISSUE TITLE
Issue when using local/on-premises AWS IoT Shadow platforms (like Greengrass)

### ISSUE THREAD

**Issue by ghost** [mongoose-os-libs/aws#11]
The AWS shadow library doesn't work when connecting to a local/on-premises AWS IoT Shadow platforms (like Greengrass).

The error I found in my console log: `"MQTT is not configured for AWS, not initialising shadow"`

My `mos.yml` configuration section:
```
config_schema:
  - ["mqtt.enable", true]
  - ["mqtt.server", "192.168.1.100:8883"]
  - ["mqtt.ssl_ca_cert", "ca.pem"]
  - ["mqtt.user", "..."]
  - ["mqtt.pass", "..."]
  - ["shadow.lib", "aws"]
  - ["aws.greengrass.enable", true]
```

**Comment by ghost** [comment]
Fixed, see #10 (as per discussion with @cpq)

**Comment by ghost** [comment]
Pull request #10 has been integrated, so the issue is fixed.
